### PR TITLE
Update microsoft-intune-app-linux.md

### PR DESCRIPTION
**Pull request by msbemba**
Added cmd to allow the Company Portal to read the disk encryption

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4040

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/user-help/microsoft-intune-app-linux.md [hunk 1]
```diff
@@ -66,9 +66,13 @@ Run the following commands in a command line to manually install the Intune app
 
     `$ sudo apt update` 
 
-    `$ sudo apt install intune-portal` 
+    `$ sudo apt install intune-portal`
 
-4. Reboot your device.   
+4. Run the following command to allow the Company Portal to read the disk encryption
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/microsoft-intune-app-linux.md:71]:
```suggestion
4. Run the following command to allow the Company Portal to read the disk encryption:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and signoff this PR. Thanks

**Comment by lenewsad** [comment]
Hi @msbemba @yogkumgit, thanks for making this change--asking PM to review.

**Comment by siimsoni** [comment]
Hi,

please take a careful consideration prior to adding users to groups, as it can have security implications:

> The group disk can be very dangerous, since hard drives in /dev/sd* and /dev/hd* can be read and written bypassing any file system and any partition, allowing a normal user to disclose, alter and destroy both the partitions and the data of such drives without root privileges. Users should never belong to this group.

https://wiki.debian.org/SystemGroups

File: memdocs/intune/user-help/microsoft-intune-app-linux.md [hunk 2]
```diff
@@ -66,9 +66,13 @@ Run the following commands in a command line to manually install the Intune app
 
     `$ sudo apt update` 
 
-    `$ sudo apt install intune-portal` 
+    `$ sudo apt install intune-portal`
 
-4. Reboot your device.   
+4. Run the following command to allow the Company Portal to read the disk encryption:
+   
+    `$ sudo usermod -a -G disk <USER_ACCOUNT>`
```
Comment (Foorack) [inline memdocs/intune/user-help/microsoft-intune-app-linux.md:73]: I have **security concerns** regarding this change.

image

Recommending to add the local user to group `disk` is equivalent of giving that user SuperAdmin (root) access.

**Review by Foorack — CHANGES_REQUESTED** [review]

**Comment by lenewsad** [comment]
Hi @msbemba, we can close this PR. I have been unable to get an answer from the product team that confirms this change is needed.

**Pull request closed without merging**